tests/integration/README.md

```diff
@@ -1,84 +1,7 @@
 # Sail Operator Integration Test
 
-This integration test suite utilizes Ginkgo, a testing framework known for its expressive specs (reference: https://onsi.github.io/ginkgo/). The setup for the test run is similar to the upstream Istio integration tests:
-* In the case of kind execution, it relies on the upstream script `kind_provisioner.sh` and `integ-suite-kind.sh`, which are copied from the `github.com/maistra/istio` repository to set up the kind cluster used for the test.
-* In the case of OCP execution, it relies on the `inter-suite-ocp.sh` and `common-operator-integ-suite` scripts to setup the the OCP cluster to be ready for the test.
+This integration test suite utilizes Ginkgo, a testing framework known for its expressive specs (reference: https://onsi.github.io/ginkgo/). 
 
-## Pre-requisites
+## WIP
```

Comment:
What do you mean? There's no integration tests yet?

Reply:
Yes, we just split the test suites: integration test are related to the api test that Marko is adding and e2e test are related to the new test that we just add. I put WIP here because I need to check all the information in the api integration test to write the proper readme file